tests/lib/rules/padding-line-between-statements.js

```diff
@@ -2729,7 +2729,7 @@ ruleTester.run("padding-line-between-statements", rule, {
             parserOptions: { ecmaVersion: 2022 }
         },
         {
-            code: "class C { static { 'use strict'; let x; } }", // 'use strict'; is "espression", because class static blocks don't have directives
+            code: "class C { static { 'use strict'; let x; } }", // 'use strict'; is "expression", because class static blocks don't have directives
```

Comment:
Not sure if this was intentional since it is in a quote

Reply:
This was an unintentional typo. It refers to statement type `"expression"` that can be specified in the [configuration](https://eslint.org/docs/rules/padding-line-between-statements#rule-details) for this rule.

The change is good.